politeiawww/legacy/comments/process.go

```diff
@@ -314,26 +315,91 @@ func (c *Comments) processVotes(ctx context.Context, v v1.Votes) (*v1.VotesReply
 	cv := convertCommentVotes(votes)
 
 	// Populate comment votes with user data
-	uid, err := uuid.Parse(v.UserID)
+	err = c.commentVotesPopulateUserData(cv, v.UserID)
 	if err != nil {
 		return nil, err
 	}
-	u, err := c.userdb.UserGetById(uid)
-	if err != nil {
-		return nil, err
-	}
-	commentVotePopulateUserData(cv, *u)
-
-	// Sort comment votes by timestamp from newest to oldest.
-	sort.SliceStable(cv, func(i, j int) bool {
```

Comment:
Good call moving this sorting into the plugin.  That's the correct place for it.